pkg/acs/calacs/acsrej/rej_command.c

```diff
@@ -47,12 +47,27 @@ int newpar[];       o: array of parameters set by the user
         List all options for user
     */
     if (argc < 3) {
-        syntax_error ("acsrej input output [-t] [-v]");
-        printf("                    [-shadcorr] [-crmask] [-newbias] \n ");
+        syntax_error ("acsrej.e input output [-t] [-v]");
+        printf("                    [-shadcorr] [-crmask] [-readnoise_only] \n");
```

Comment:
This needs a corresponding change over at ACSTOOLS (e.g., https://github.com/spacetelescope/acstools/blob/master/acstools/pars/acsrej.cfg).

Reply:
My input: I would argue that this 1-1 API mapping creates an unnecessary dependency. I would state that it is better to use the, recently added, 'transparent window' `exe_args` and then just make this issue one of documentation only (since the ACS HSTCAL docs exist as doc strings in acstools). However, I understand that since the other parameters are mirrored the precedence is already set and it's already too late.